myscripts.js

```diff
@@ -0,0 +1,173 @@
+const toDoObjectsArray =[]
+document.getElementById('mainSubmitButton').setAttribute('onclick','newToDoItem()')
```

Comment:
no need for a string inside `setAttribute`. I more than sure that you can use `setAttribute('onclick', newToDoItem)`